rwandaemr-installer/src/main/resources/scripts/download-distribution.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/bash -eu
 
-ARTIFACT_ID="$1"
-VERSION="$2"
-TARGET_DIR="$3"
+## load variables from the vars.txt file
+source vars.txt
+
+ARTIFACT_ID="${distribution}"
+VERSION="${version}"
+
+if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
```

Comment:
Don't maintain this if clause in multiple places.  Encapsulate this in a single place.